orttraining/orttraining/core/optimizer/compute_optimizer/padding_elimination.cc

```diff
@@ -404,21 +423,6 @@ Status PaddingElimination::ApplyImpl(Graph& graph, bool& modified, int graph_lev
         LOG_DEBUG_INFO(logger, "not find PythonOp of flagPaddingElimination after embedding node");
```

Comment:
shall we record the pythonop node pointer here, then we don't need loop later to find it again.

Reply:
The loop is in case if not find valid embedding_node and not get the pythonop.